include/itkStructurePreservingColorNormalizationFilter.h

```diff
@@ -269,12 +311,11 @@ class StructurePreservingColorNormalizationFilter : public ImageToImageFilter< T
 
   // These members are for the purpose of caching results for use the
   // next time the pipeline is run.
-  itk::ModifiedTimeType m_ParametersMTime;
-  const ImageType *m_inputPtr;
-  TimeStamp m_inputTimeStamp;
+  ModifiedTimeType m_ParametersMTime;
+  const ImageType *m_input;
   CalcMatrixType m_inputH;
   CalcRowVectorType m_inputUnstainedPixel;
-  const ImageType *m_referPtr;
+  const ImageType *m_refer;
```

Comment:
Per ITK naming conventions `m_refer` -> `m_Reference` (CamelCase, full names). `refer` in the variables below should also be updated.

Reply:
Several variable names and member names failed to follow the convention; fixing that.